### ISSUE TITLE
Song info not appearing from Spicetify after playing a song from browser

### ISSUE THREAD

**Issue by tiergagreit** [Meti0X7CB/WinTaskical#4]
I use spicetify and already have webnowplaying installed on my browser. At first when I played a song in spotify desktop it works well. Then I closed the app, opened youtube from browser. The title and the channel and the thumbnail appeared, normal. But when I played song from spotify again, the info doesn't appear anymore. Now it only shows song info from the browser. Is there any way to fix it? (Squareplayer also has the same issue)

**Comment by Meti0X7CB** [comment]
so u closed all browser instances and played spicetify and it doesnt show anything?

**Comment by tiergagreit** [comment]
yes. Somehow the song info reappeared when I install/reinstall spiceitfy custom_apps from powershell, but the issue is still not fixed. Though there is no problem when I played spicetify while browsing youtube (the skin still displays song info from spicetify)

**Comment by Meti0X7CB** [comment]
please test with new version
